src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.cs

```diff
@@ -34,10 +33,15 @@ public partial class ShadowContainer : ContentControl
 
 	private readonly SerialDisposable _eventSubscriptions = new();
```

Comment:
can we group disposables, references and states together?
delete line 40, line 35

Reply:
yup